### PR TITLE
[Web-App] Remove supertokens

### PR DESCRIPTION
**Pull request by Maks19**
**Description:** Remove supertokens

**Notion Task:** https://www.notion.so/saladtech/Web-App-remove-SuperTokens-2992c392035c405194b6e56cb527ba19

### CODE REVIEW

File: packages/web-app/src/axiosFactory.tsx [hunk 1]
```diff
@@ -32,10 +31,10 @@ const shouldRetryDownload = (error: any): boolean => {
 
 export const createClient = (): AxiosInstance => {
   let httpClient = axios.create({
+    withCredentials: true,
```
Comment (vitto-moz) [inline packages/web-app/src/axiosFactory.tsx:34]: is this for third-party requests? 
do we use axios to request third-party services?

Comment (Maks19) [reply packages/web-app/src/axiosFactory.tsx:34]: It was done for requests should be made using credentials such as cookies, authorization headers or TLS client certificates
Without this parameter, our axios requests won't include `auth` token

Comment (seniorquico) [reply packages/web-app/src/axiosFactory.tsx:34]: Is this httpClient used to make requests to any API other than ours? If yes, then ideally this value is set true only for those requests.

File: packages/web-app/src/axiosFactory.tsx [hunk 2]
```diff
@@ -45,12 +44,8 @@ export const createClient = (): AxiosInstance => {
...
       try {
         if (error.response.status === 401 && error.config.baseURL === config.apiBaseUrl) {
-          SuperTokens.doesSessionExist().then((result) => {
-            if (!result) {
-              const store = getStore()
-              store.auth.setIsAuthenticated(false)
-            }
-          })
+          const store = getStore()
+          store.auth.setIsAuthenticated(false)
```
Comment (seniorquico) [inline packages/web-app/src/axiosFactory.tsx:48]: Is the httpClient this interceptor attached to used to make requests to any API other than ours? If yes, then a failing request to another API will cause the app to logout. The removed SuperTokens session check first verified that it was actually expired before causing the app to logout.

**Review by seniorquico — CHANGES_REQUESTED** [review]

File: packages/web-app/src/axiosFactory.tsx (hunk 1 above)
Comment (Maks19) [reply packages/web-app/src/axiosFactory.tsx:34]: We're not making any third-party requests with an axios and the `httpClient` is axios instance so it's not making requests to any API other than ours

File: packages/web-app/src/axiosFactory.tsx (hunk 2 above)
Comment (Maks19) [reply packages/web-app/src/axiosFactory.tsx:48]: We're not making any third-party requests with an axios so this interceptor is not attached to requests to any API other than ours

**Review by seniorquico — APPROVED** [review]

**Review by vitto-moz — APPROVED** [review]

**Pull request merged**